apps/docs/src/guide/predicates/methods.md

```diff
@@ -0,0 +1,55 @@
+# Interacting With Predicates
+
+The `Predicate` class extends the [`Account`](https://docs.fuel.network/docs/fuels-ts/account/) class, inheriting all its methods. Therefore, there are multiple ways to interact with predicates, but broadly speaking, we can think about three:
+ - `Checking Balances`
+ - `Transactions`
+ - `Transfers`
+
+
+## Checking Balances
+
+### `getBalances`
+
+This will return the balances of all assets owned by the predicate.
+
+See also: [Checking Wallet Balances](https://docs.fuel.network/docs/fuels-ts/wallets/checking-balances/#getting-a-wallets-balance)
```

Comment:
I realized my first comment may have been misleading.

What I meant was to remove this section entirely, and then — see the next suggestion.

```suggestion
```

Reply:
Dismissing my own suggestion. 🤦‍♂️